resources/eventing/profile-production.yaml

```diff
@@ -14,15 +14,15 @@ controller:
       cpu: 500m
       memory: 1Gi
     requests:
-      cpu: 400m
+      cpu: 10m
```

Comment:
Isn't `10m` too less for production profile?

I deployed this PR with production profile and ran some tests:

With 50 events/sec:
![image](https://user-images.githubusercontent.com/9897945/173576249-e8a486f4-2ca4-46a8-8fa9-7fa56e207454.png)

With 10 events/sec:
![image](https://user-images.githubusercontent.com/9897945/173576321-869149c3-387c-45a1-9d13-63589ec2d75b.png)

You can see that `10m` was not enough even for 10events/sec. 
IMO, the requested CPU should be at-least enough to handle minimum production workload.

I would say, that we use atleast `150m` for production profile, so it can support 50events/sec using the requested CPU.

Reply:
@mfaizanse 🤔 we are talking about requested cpu time - which should rather correspond to idle/startup state.
What we see is that an iddle kyma runtime suffers from huge overbooking of the CPU time across all components.

When eventing is under load it uses more  - it is seen in you screenshots, ie:
nats requests `0.01` from k8s scheduler at the beginning.
When the load starts - the actual  CPU usage rises up to `0.15`, that corresponds to 1450% of the initially requested amount.. but still.. its only 30% of the limit!
The pod is free to rise cpu consumption until its limits (the limits where not changed by the PR).
IMO there is no justification to overbook cpu time "just in case".

Reply:
Question: When a pod is scheduled by K8s on a node, does it also check if the pod will be able to reach it limits on that node or not? For example, if a node has only `50m` CPU free (or unreserved) and the `eventing-controller` pod is scheduled on that node, will the eventing-controller pod be able to scale further up 50m? or what will happen in this case?

Reply:
in my cluster I also apply load to eventing component
The actual load ( cpu usage ) is not affected by the reduced cpu request time.
It can freely go up until limit

![Screenshot 2022-06-14 at 15 24 42](https://user-images.githubusercontent.com/6783567/173588707-4781e6ab-4b44-4f0e-987f-b1614e9c9dbb.png)